src/dbos-executor.ts

```diff
@@ -121,6 +122,7 @@ export interface DBOSConfig {
   databaseUrl?: string;
   userDatabaseClient?: UserDatabaseName;
   userDatabasePoolSize?: number;
+  enableUserDatabase?: boolean;
```

Comment:
I don't understand the need for this field. It will always be true if userDatabaseClient is set and always false otherwise.